spec/factories.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :user do |u|
+    u.sequence(:first_name) { |n| "User#{n}"}
```

Comment:
Czy `first_name` jest wymagane przez model? Jeśli nie to raczej bym proponował go nie ustawiać - reguła kciuka mówi żeby faktory ustawiało absolutnie minimalny wymagany zestaw danych. Poza względami praktycznymi (np. mniejsza ilość danych zapisywana do bazy danych) ma to jeszcze jeden powód: będziesz miała w przyszłości faktycznych użytkowników którzy tego nie ustawią i testy mogą Ci czegoś nie wyłapać (np. index wyświetla gdzieś `first_name`, ale że użytkownik nie ustawił to coś się wysypuje).